sustc-api/src/main/java/io/sustc/service/RecommenderService.java

```diff
@@ -52,11 +52,11 @@ public interface RecommenderService {
 
     /**
      * Recommends videos for a user, restricted on their interests.
-     * The user's interests are defined as the videos that the both user's followers and his/her followees have watched,
+     * The user's interests are defined as the videos that the both one of the user's followers and one of his/her followees have watched,
```

Comment:
($\bigcup_{f \in followers}$ Vid<sub>f</sub>) $\cup$ ($\bigcup_{f \in followees}$ Vid<sub>f</sub>)

or

$\bigcup_{f \in friends}$ Vid<sub>f</sub>